contents/docs/llm-analytics/installation/_snippets/llms-sdks-callout.mdx

```diff
@@ -0,0 +1,9 @@
+import { CalloutBox } from 'components/Docs/CalloutBox'
+
+<CalloutBox type="fyi" icon="IconInfo" title="Note">
+
+These SDKs _do not_ proxy your calls, they only fire off an async call to PostHog in the background to send the data.
+
+You can also use LLM analytics with other SDKs or our API, but you will need to capture the data manually via the capture method. See schema in the [manual capture section](/docs/llm-analytics/manual-capture) for more details.
```

Comment:
Is there privacy concerns to address somewhere about this? I can think how one might be concerned about what we collect, what this SDk docs, do we access their api keys etc.